build_docs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+export JRUBY_OPTS="-J-Xmx2g"
+export GRADLE_OPTS="-Xmx2g -Dorg.gradle.daemon=false"
+
+export PATH="$HOME/.rbenv/bin:$PATH"
+eval "$(rbenv init -)"
+rbenv local jruby-9.1.12.0
+
+echo "Cloning 'logstash-docs'"
+git clone --depth 1 git@github.com:elastic/logstash-docs.git -b $branch_specifier
+
+echo "Cloning 'logstash'"
+git clone --depth 1 git@github.com:elastic/logstash.git -b $branch_specifier
+
+echo "Cloning 'docs'"
+git clone --depth 1 git@github.com:elastic/docs.git
+
+cd logstash
+
+patch -p1 <../docs-tools/logstash/docs.patch
```

Comment:
nitpick: prefer long-form options in scripts for readability

~~~ sh
patch --strip=1 < ../docs-tools/logstash/docs.patch
~~~